js/exporter/jspdf/v3/export_data_grid_3.js

```diff
@@ -103,10 +103,28 @@ function exportDataGrid(doc, dataGrid, options) {
                 x: doc.internal.pageSize.getWidth() - options.margin.right
             };
             const onSeparateRectHorizontally = (sourceRect, leftRect, rightRect) => {
-                const newRectCellInfo = Object.assign({}, sourceRect.sourceCellInfo, { text: '', debugSourceCellInfo: sourceRect.sourceCellInfo });
+                let leftRectTextOptions = {};
+                let rightRectTextOptions = {};
+                const isTextNotEmpty = isDefined(sourceRect.sourceCellInfo.text) && sourceRect.sourceCellInfo.text.length > 0;
+                if(isTextNotEmpty) {
+                    const isTextWidthGreaterThanRect = doc.getTextWidth(sourceRect.sourceCellInfo.text) > leftRect.w;
+                    const isHorizontalAlignSpecified = isDefined(sourceRect.sourceCellInfo.horizontalAlign) && sourceRect.sourceCellInfo.horizontalAlign !== 'left';
+                    if(isTextWidthGreaterThanRect || isHorizontalAlignSpecified) {
+                        const leftTextOffset = sourceRect.sourceCellInfo._textOffset ?? { left: 0, top: 0 };
```

Comment:
I cannot find where the '_textOffset.top' is used? Is it used at all? Do we need 'top: 0'?

Reply:
Yes, at the current time the '_textOffset.top' is not used. This field i'm added on the future. For vertical text splitting.
Ok, I'm will remove this field from this place, because it ont used at the moment, and add it when it will be neccessary.